scripts/deploy.sh

```diff
@@ -112,24 +120,23 @@ fi
 # Apply host security hardening with openstack-ansible-security
 # The is applied as part of setup-hosts.yml
 if [[ "$DEPLOY_HARDENING" == "yes" ]]; then
-  if grep -q '^apply_security_hardening:' /etc/openstack_deploy/user_variables.yml
-  then
-    sed -i "s/^apply_security_hardening:.*/apply_security_hardening: true/" /etc/openstack_deploy/user_variables.yml
+  if grep -q '^apply_security_hardening:' /etc/openstack_deploy/user_osa_variables_defaults.yml; then
+    sed -i "s/^apply_security_hardening:.*/apply_security_hardening: true/" /etc/openstack_deploy/user_osa_variables_defaults.yml
   else
-    echo "apply_security_hardening: true" >> /etc/openstack_deploy/user_variables.yml
+    echo "apply_security_hardening: true" >> /etc/openstack_deploy/user_osa_variables_defaults.yml
   fi
 else
-  if grep -q '^apply_security_hardening:' /etc/openstack_deploy/user_variables.yml
-  then
-    sed -i "s/^apply_security_hardening:.*/apply_security_hardening: false/" /etc/openstack_deploy/user_variables.yml
+  if grep -q '^apply_security_hardening:' /etc/openstack_deploy/user_osa_variables_defaults.yml; then
+    sed -i "s/^apply_security_hardening:.*/apply_security_hardening: false/" /etc/openstack_deploy/user_osa_variables_defaults.yml
   else
-    echo "apply_security_hardening: false" >> /etc/openstack_deploy/user_variables.yml
+    echo "apply_security_hardening: false" >> /etc/openstack_deploy/user_osa_variables_defaults.yml
   fi
 fi
 
 # ensure all needed passwords and tokens are generated
-./scripts/pw-token-gen.py --file /etc/openstack_deploy/user_secrets.yml
+./scripts/pw-token-gen.py --file /etc/openstack_deploy/user_osa_secrets_defaults.yml
 ./scripts/pw-token-gen.py --file $RPCD_SECRETS
```

Comment:
pw-token-gen.py is used to set a password for a variable defined in a file. If this script isn't run against the overrides files it doesn't help anyone because now support just have to remember to do it manually.